src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/ServiceDocumentFormatter.cs

```diff
@@ -2,309 +2,303 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Collections.ObjectModel;
+using System.Diagnostics;
 using System.Runtime.Serialization;
-using System.Xml.Serialization;
-using System.Collections.Generic;
 using System.Xml;
-using System.Runtime.CompilerServices;
 
 namespace System.ServiceModel.Syndication
 {
     [DataContract]
     public abstract class ServiceDocumentFormatter
     {
-        private ServiceDocument _document;
-
         protected ServiceDocumentFormatter()
         {
         }
+
         protected ServiceDocumentFormatter(ServiceDocument documentToWrite)
         {
-            if (documentToWrite == null)
-            {
-                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("documentToWrite");
-            }
-            _document = documentToWrite;
+            Document = documentToWrite ?? throw new ArgumentNullException(nameof(documentToWrite));
         }
 
-        public ServiceDocument Document
-        {
-            get { return _document; }
-        }
+        public ServiceDocument Document { get; private set; }
 
```

Comment:
This is technically an api change. Someone reflecting on ServiceDocumentFormatter will now see a private setter which isn't there on the contract.

Reply:
I *think* we’ve taken these sort of changes before? Will gladly revert though? @stephentoub May be able to comment

Ether way we should update https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md#property-field-parameter-and-return-values

Reply:
Under the same logic removing/adding something like an internal field or removing dead code in an internal method may be considered breaking

Reply:
We do not consider additions/changes/removals to private APIs as being breaking.

Reply:
@stephentoub, when DataContractSerializer is involved, it can be a breaking change as it will use a private setter. Many of the classes in this project are design to be serialized with DataContractSerializer so while this isn't normally a breaking change, with this specific project it can be.

Reply:
@hughbe, adding or removing an internal or private field in a class attributed with `[Serializable]` absolutely is a breaking change. When a class is intended to be serialized, things which you wouldn't normally consider to be a breaking change now become a breaking change as it modifies how things get serialized.